models/src_greenhouse.yml

```diff
@@ -487,6 +488,17 @@ sources:
       - name: tag
         identifier: "{{ var('greenhouse_tag_identifier', 'tag')}}"
         description: Table storing candidate tags. 
+        freshness: null
```

Comment:
Why do some of these tables have freshness disabled? Wondering if I need this for my PR.

Reply:
Ah, it was because of this note in ticket:

> Be sure to include both renames in the src.yml file and let’s remove our freshness tests from these models by setting them as null so we don’t run into any issues in the future. Additionally I don’t imagine users and tags being frequently updated tables so this should be appropriate.